burner-ui/src/components/BalanceItem/BalanceItem.tsx

```diff
@@ -3,6 +3,39 @@ import { Asset } from '@burner-wallet/assets';
 import { Card, Box, Flex } from 'rimble-ui';
 import styled from 'styled-components';
 import Text from '../Text';
+import { Eth, Xdai, Dai } from '../TokenIcons';
+
+interface BalanceItemProps {
+  asset: Asset | string;
```

Comment:
Seems like some of these problems may be coming from accepting both of these as types, this should probably just be Asset right?

Reply:
yeah you're right

Reply:
Updated 👍